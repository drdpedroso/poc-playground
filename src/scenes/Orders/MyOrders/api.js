import { gql } from 'apollo-client-preset/lib/index'

export const AllOrder = gql`
query getOrders ($filter: OrderFilter!, $userToken: String!) {
  allOrder (filter: $filter, userToken: $userToken) {
    orderNumber
    pocDocument
    customerEmail
    customerName
    deliveryType
    expiringTime
    pocName
    orderRating
    orderDate
    orderDetails
  }
}`
