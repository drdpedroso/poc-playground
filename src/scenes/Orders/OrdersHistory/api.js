import { gql } from 'apollo-client-preset/lib/index'

export const AllOrdersQuery = gql`query getAllOrders ($search: String, $skip: Int, $first: Int, $userToken: String!) {
  allOrder (search: $search, skip: $skip, first: $first, userToken: $userToken) {
    orderNumber
    customerEmail
    customerName
    deliveryType
    expiringTime
    pocName
    orderRating
    orderDate
    orderDetails
    statusHistory {
      status
      comments
    }
  }
}`
