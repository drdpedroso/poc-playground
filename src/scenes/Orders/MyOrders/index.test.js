import React from 'react'
import { MyOrders } from './index'
import renderer from 'react-test-renderer'
import { AllOrder } from './api'

describe('MyOrders ', () => {
  it('should render correctly', () => {
    const component = renderer.create(<MyOrders />)
    expect(component).toMatchSnapshot()
  })
})
