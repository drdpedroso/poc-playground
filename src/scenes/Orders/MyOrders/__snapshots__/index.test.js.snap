// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyOrders  should render correctly 1`] = `
<div>
  Leke
</div>
`;
